app/_data/docs_nav_kuma_dev.yml

```diff
@@ -805,3 +807,150 @@ items:
             url: "/contribute/introduction/#gui-code"
           - text: Testing unreleased versions
             url: "/contribute/introduction/#testing-unreleased-versions"
+unlisted:
+  - url: /
+  - url: /documentation/dependencies/
+  - url: /documentation/introduction/
+  - url: /policies/matching/
```

Comment:
This page was just forgotten to be removed when it was replaced by `/policies/targetRef`